[JIT] Make IRParser use op schema

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32938 [WIP][quantization] Use torchbind for Linear PackedParams
* #32977 [JIT] Resolve custom classes in source importer
* #32948 [c10] Allow taking a std::tuple as arg
* #32878 [JIT] Fix python pickle serialization for torchbind
* **#32854 [JIT] Make IRParser use op schema**
* #32833 [JIT] Trace uses of torchbind classes as module attributes

Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

Commit:
[JIT] Make IRParser use op schema

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"




[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"




[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 9c7f2d09:


**None of the build failures appear to be your fault**.

* **1/1** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=9c7f2d09518a6ac5ddafca63472249d73930822e).

### :snowflake: 1 failure recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4387338) binary_linux_libtorch_2_7m_cpu_gcc5_4_cxx11-abi_shared-with-deps_build (1/1)
**Step:** "Install unbuffer and ts" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=93588835) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=93588835)) :snowflake:
<details>
<summary>
<code>CondaHTTPError: HTTP 502 BAD GATEWAY for url <https://conda.anaconda.org/eumetsat/linux-64/current_repodata.json></code>
</summary>

```
A 500-type error (e.g. 500, 501, 502, 503, etc.) indicates the server failed to 
fulfill a valid request.  The problem may be spurious, and will resolve itself if you 
try your request again.  If the problem persists, consider notifying the maintainer 
of the remote server. 
 
 
+ sleep 8 
+ conda install -y -c eumetsat expect 
Collecting package metadata (current_repodata.json): - \ | / - \ | / - \ failed 
 
CondaHTTPError: HTTP 502 BAD GATEWAY for url <https://conda.anaconda.org/eumetsat/linux-64/current_repodata.json> 
Elapsed: 00:00.053848 
CF-RAY: 55d84f156c16cf1c-IAD 
 
A remote server error occurred when trying to retrieve this URL. 
 
A 500-type error (e.g. 500, 501, 502, 503, etc.) indicates the server failed to 
fulfill a valid request.  The problem may be spurious, and will resolve itself if you 
try your request again.  If the problem persists, consider notifying the maintainer 
of the remote server. 
 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]

Commit:
Update on "[JIT] Make IRParser use op schema"


Differential Revision: [D19656881](https://our.internmc.facebook.com/intern/diff/D19656881)

[ghstack-poisoned]
